torchmetrics/utilities/data.py

```diff
@@ -248,3 +248,10 @@ def _bincount(x: Tensor, minlength: Optional[int] = None) -> Tensor:
         return output
     else:
         return torch.bincount(x, minlength=minlength)
+
+
+def xlogy(x: Tensor, y: Tensor) -> Tensor:
```

Comment:
maybe here https://github.com/PyTorchLightning/metrics/blob/master/torchmetrics/utilities/compute.py
or find some better name?

Reply:
fine by me, have moved it to the file and also renamed (+added doctest)